modules/ztp-add-local-reg-for-sno-duprofile.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-advanced-policy-config.adoc
+
+:_module-type: CONCEPT
+[id="ztp-add-local-reg-for-sno-duprofile_{context}"]
+= Configuring the Image Registry Operator for local caching of images
+
+{product-title} manages image caching using a local registry. In edge computing use cases, clusters are often subject to bandwidth restrictions when communicating with centralized image registries, which might result in long image download times.
+
+Long download times are unavoidable during initial deployment. Over time, there is a risk that CRI-O will erase the `/var/lib/containers/storage` directory in the case of an unexpected shutdown.
+
+To address long image download times, you need a local image registry on {sno}. This is useful in Edge scenarios where {sno} is deployed on the Far Edge. {sno} cannot access the internet, just its own resources and eventually the internal network.
```

Comment:
I can see whaty you are getting at, but this sentence is a bit confusing. "{sno} cannot access the internet, just its own resources and eventually the internal network." What about:

```suggestion
To address long image download times, you can create a local image registry on the remote managed cluster using GitOps ZTP. This is useful in Edge computing scenarios where clusters are deployed on the far edge of the network.
```

Reply:
Done.